rpm/stackdriver-agent.spec

```diff
@@ -296,7 +298,7 @@ fi
 
 %files
 %defattr(-, root, root, -)
-%config %{_sysconfdir}/collectd.conf
+%config %{_confdir}/collectd.conf
 %config(noreplace) %{_sysconfdir}/collectd.d/
```

Comment:
Why don't we also put `collectd.d` under `/etc/stackdriver`?

Reply:
+1

Reply:
As discussed offline, let's look in both the new path and the old path (the new one taking precedence, so look in the old one first).

Reply:
Done.